Modules/IO/PhilipsREC/src/itkPhilipsRECImageIO.cxx

```diff
@@ -29,7 +29,7 @@
 #include "itkIOCommon.h"
 #include "itkByteSwapper.h"
 #include "itkMetaDataObject.h"
-#include "itkSpatialOrientationAdapter.h"
+#include "itkDICOMOrientation.h"
```

Comment:
The idea is to rename this without DICOM, correct?

Reply:
Thanks. I don't think this module is enabled by default.

Reply:
Done